src/storage/raw/store.rs

```diff
@@ -223,6 +223,53 @@ impl<'a, S: Snapshot> RawStoreInner<S> {
         Ok(pairs)
     }
 
+    pub async fn forward_raw_scan_v2(
```

Comment:
I think we need to skip entries: 1. without the largest timestamp of the same user key, and 2. logical deleted.

Reply:
now i implement the raw_get_key_value/ forward_raw_scan/reverse_raw_scan operation in RawStoreV2 to have unifued view. Same key is skipped, and deletion can only be judged after PR#12090 merged.

Reply:
Define apiv2 RawStore as RawStoreInner<RawEncodeSnapshot<RawMvccSnapshot, APIV2>>, RawMvccSnapshot process mvcc of keys and RawEncodeSnapshot process meta in value